routes/web.php

```diff
@@ -20,3 +21,10 @@
 Auth::routes();
 
 Route::get('/home', [HomeController::class, 'index'])->name('home');
+
+Route::get('admin/users', [AdminController::class, 'index'])->name('admin.index');
```

Comment:
Ik heb bij mijn gedeelte een webpagina url met /admin... voor de confidants. We moeten even bespreken hoe we dit gaan doen, want ik zie nu dat jij de url /admin... gebruikt voor de admin (Ed).